src/Entity/Transformer.php

```diff
@@ -179,7 +179,7 @@ public function loadBlock(array $content, PageInterface $page): PageBlockInterfa
             $block->setName($content['name']);
         }
 
-        $block->setEnabled($content['enabled']);
+        $block->setEnabled(\in_array($content['enabled'], ['1', true], true));
```

Comment:
and whats happen when it's '0'?

Reply:
ahh ok, won't have '0' into the array and it will be false!

it's not really explicit that is working for '1' and '0', But I got it!


maybe should be good create a test with `0`

Reply:
Unless someone has a snapshot with 1|0 as integer, I prefer to not add things for "what if".

Reply:
I mean create a test with `'0'` as string

Reply:
Added '0' test case.